marketplace/contracts/DataExchange.sol

```diff
@@ -8,25 +8,25 @@ pragma solidity 0.5.7;
  *         For more information: https://wibson.org
  */
 contract DataExchange {
-  event NotaryRegistered(address indexed notary, string notaryUrl);
-  event NotaryUpdated(address indexed notary, string oldNotaryUrl, string newNotaryUrl);
-  event NotaryUnregistered(address indexed notary, string oldNotaryUrl);
-  event DataOrderCreated(uint256 indexed orderId, address indexed buyer);
-  event DataOrderClosed(uint256 indexed orderId, address indexed buyer);
-
-  struct DataOrder {
-    address buyer;
-    string audience;
-    uint256 price;
-    string requestedData;
-    bytes32 termsAndConditionsHash;
-    string buyerUrl;
-    uint32 createdAt;
-    uint32 closedAt;
-  }
-
-  DataOrder[] private dataOrders;
-  mapping(address => string) private notaryUrls;
+    event NotaryRegistered(address indexed notary, string notaryUrl);
+    event NotaryUpdated(address indexed notary, string oldNotaryUrl, string newNotaryUrl);
+    event NotaryUnregistered(address indexed notary, string oldNotaryUrl);
+    event DataOrderCreated(uint256 indexed orderId, address indexed buyer);
+    event DataOrderClosed(uint256 indexed orderId, address indexed buyer);
+
+    struct DataOrder {
+        address buyer;
+        string audience;
+        uint256 price;
+        string requestedData;
+        bytes32 termsAndConditionsHash;
+        string buyerUrl;
+        uint32 createdAt;
+        uint32 closedAt;
+    }
+
+    DataOrder[] internal dataOrders;
+    mapping(address => string) internal notaryUrls;
```

Comment:
I would just leave these two as `public`. Anyone can use either the getters or the attributes directly.

Reply:
Sure! I was not sure why we declared them as private on the first place, so I didn't want to change its behavior (that much).